### PR TITLE
Fix windows test example

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

- Check dataset before extract
- Use `setuptools<72.1` to avoid issue on build torch extension

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
windows/job/test-examples/244/

File: tests/cross_fw/examples/run_example.py [hunk 1]
```diff
@@ -250,8 +250,9 @@ def quantization_aware_training_torch_anomalib(data: Union[str, None]):
     if data is not None:
         dataset_path.mkdir(parents=True, exist_ok=True)
         tar_file_path = Path(data) / mvtec.DOWNLOAD_INFO.url.split("/")[-1]
-        with tarfile.open(tar_file_path) as tar_file:
-            tar_file.extractall(dataset_path)
+        if not tar_file_path.exists():
```
Comment (alexsu52) [inline tests/cross_fw/examples/run_example.py:253]:
```suggestion
        if tar_file_path.exists():
```

Comment (AlexanderDokuchaev) [reply tests/cross_fw/examples/run_example.py:253]: ?

**Review by MaximProshin — COMMENTED** [review]
@AlexanderDokuchaev , please make sure that test-examples on Ubuntu are green with your changes.

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**